### PR TITLE
Node 6.2.0 release Testnet

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Inform users about the contents of the node release.

## Changes

- Release notes
- Downloads
- Previous node downloads
- Remove old grpc doc and update v2 doc for all SDKs and to remove references to the old doc.
- Update grpc port of the concordium testnet node

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

**Comment by dg-concordium** [comment]
Note that the changes to the client commands are handled in a separate PR.

File: source/mainnet/net/release-notes/release-notes-lp.rst [hunk 1]
```diff
@@ -985,12 +985,22 @@ Mainnet
 Testnet
 -------
 
-    October 16, 2023
+    November 28, 2023
 
-    Version 6.1.7 adds load shedding for gRPC v2. This helps with protecting the node in case of high number of concurrent requests since they are now dropped immediately as opposed to queued.
+    Version 6.2.3 deprecates the gRPC v1 API. This removes the configuration options ``CONCORDIUM_NODE_RPC_SERVER_PORT``, ``CONCORDIUM_NODE_RPC_SERVER_ADDRESS``, ``CONCORDIUM_NODE_RPC_SERVER_TOKEN``, ``CONCORDIUM_NODE_DISABLE_RPC_SERVER_NODE_ENDPOINTS`` and their command line equivalents. An additional health-check service was also added to the V2 GRPC API. This service conforms to the `standard GRPC health service API <https://github.com/grpc/grpc-proto/blob/master/grpc/health/v1/health.proto>`__.
```
Comment (abizjak) [inline source/mainnet/net/release-notes/release-notes-lp.rst:990]:
```suggestion
    Version 6.2.3 removes the V1 gRPC API. This removes the configuration options ``CONCORDIUM_NODE_RPC_SERVER_PORT``, ``CONCORDIUM_NODE_RPC_SERVER_ADDRESS``, ``CONCORDIUM_NODE_RPC_SERVER_TOKEN``, ``CONCORDIUM_NODE_DISABLE_RPC_SERVER_NODE_ENDPOINTS`` and their command line equivalents. An additional health-check service was also added to the V2 GRPC API. This service conforms to the `standard GRPC health service API <https://github.com/grpc/grpc-proto/blob/master/grpc/health/v1/health.proto>`__.
```

File: source/mainnet/net/release-notes/release-notes-lp.rst [hunk 2]
```diff
@@ -985,12 +985,22 @@ Mainnet
...
+
+    As part of the tokenomics changes, new endpoints have been added for validators. The existing endpoints will continue to work.
```
Comment (abizjak) [inline source/mainnet/net/release-notes/release-notes-lp.rst:992]:
```suggestion
    As part of the tokenomics changes the node has new configuration options that uses the new terminology. The existing options using the legacy terminology are still supported, however they are hidden.
```

File: source/mainnet/net/release-notes/release-notes-lp.rst [hunk 3]
```diff
@@ -985,12 +985,22 @@ Mainnet
...
+
+    A ``DryRun`` endpoint has also been that allows simulating the execution of transactions.
+
+    The account map is now kept solely on disk in a separate lmdb database and it is no longer part of the internal block state database. This change results in significantly reduced resource usage for the node.
```
Comment (abizjak) [inline source/mainnet/net/release-notes/release-notes-lp.rst:996]:
```suggestion
    The account map is now kept solely on disk in a separate LMDB database and it is no longer part of the internal block state database. This change results in significantly reduced resource usage for the node.
```

**Review by abizjak — APPROVED** [review]
I think all of the JSON files under `source/mainnet/net/references/grpc-json-schema` can also be removed now.

They were JSON schemas for the V1 API.

**Comment by dg-concordium** [comment]
> Also, in the documentation we still refer to `CONCORDIUM_NODE_BAKER_CREDENTIALS_FILE` in a few places when explaining how to configure a node.
> 
> We should change that to `CONCORDIUM_NODE_VALIDATOR_CREDENTIALS_FILE` but not sure if now or for mainnet release.
> 
> I think it's not really split per network at the moment so it might be confusing to use the new one until it's also released on mainnet.

I think that those configurations are split in the documentation. I'll check.

**Review by mh-concordium — APPROVED** [review]
Looks good.

**Pull request merged**